db/controllers.js

```diff
@@ -52,3 +53,14 @@ exports.postComment = (request, response, next) => {
     })
     .catch(next);
 };
+
+exports.patchReview = (request, response, next) => {
+  const body = request.body;
+  const { review_id } = request.params;
+  Promise.all([fetchReviewById(review_id), updateReview(body, review_id)])
+    .then((results) => {
+      console.log(results);
+      response.status(201).send(results[1]);
```

Comment:
This is all good, but best practise is to send responses as object on keys which describe them, eg {updatedReview: {...the updated review}, so you could update this (it would mean changing your tests to reflect the changed structure)